yw_website/yw_website/settings.py

```diff
@@ -153,11 +153,7 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
-STATICFILES_DIRS = (
-  os.path.join(BASE_DIR, 'static/'),
-  os.path.join(BASE_DIR, 'yw_website/apps/website/static/'),
-)
+STATIC_ROOT = os.path.join(BASE_DIR, 'yw_website/apps/website/static/')
```

Comment:
This change does not make sense. Static_root is the path where all static files get put after running collectstatic, and should specifically be an empty directory. I think you are conflating this with staticfile_dirs. https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-STATIC_ROOT

Reply:
> STATIC_ROOT¶
> Default: None
> The **absolute path to the directory** where collectstatic will collect static files for deployment.
> Example: "/var/www/example.com/static/"

Not quite, `/var/www/example.com/static/` is a file path. try running the `manage.py collectstatic` command and see what happens. You'll see that a new folder is created in our repo's root directory called `staticfiles` with how our website is currently configured.

When we deploy our server to somthing like nginx, we run the collectstatic command and that will gather all static files to a single directory that is then referenced by the server. 

Additionally, I removed the `STATIC_DIRS` reference because it adds complexity, when in reality we only have one folder that we will ever want to pull css and js files from. using `STATIC_ROOT` is just simpler

Reply:
I disagree, I think this is more complex as it does not follow the Django documentation

Reply:
what part do you disagree with? all of it?

Reply:
STATICFILES_DIRS is how static files should be listed if we have multiple apps, and we do have multiple apps since we are using haystack, Django rest, and more. Also, STATIC_ROOT has nothing to do with making static files discovereable, which was what seems to be the purpose of this PR. All this will do is eliminate the discoverability of the website's static folder and change the file location of where our static files will be collected